### ISSUE TITLE
Implement New Spawning System on Existing Maps

### ISSUE THREAD

**Issue by timblegoorn** [cs481-ekh/f22-kitty-cat-studios#151]
After #149 and #150 are implemented, I will want to implement the new spawning system on the current maps.

- [x] #175 (Note: These changes were superseded by #182 due to a merge conflict with the maps)
- [x] #182
- [x] #187
